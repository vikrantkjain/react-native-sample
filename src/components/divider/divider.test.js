import React from 'react';
import renderer from 'react-test-renderer';
import Divider from './index';

test('Divider render correctly', () => {
  const tree = renderer.create(<Divider />).toJSON();
  expect(tree).toMatchSnapshot();
});
