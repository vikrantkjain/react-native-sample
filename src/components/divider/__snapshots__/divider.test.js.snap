// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider render correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#E4E4E4",
        "height": 0.8,
        "width": 750,
      },
      undefined,
    ]
  }
/>
`;
